src/backend/base/langflow/initial_setup/starter_projects/News Aggregator.json

```
@@ -243,7 +243,7 @@
                 "show": true,
                 "title_case": false,
                 "type": "str",
-                "value": "AGENTQL_API_KEY"
+                "value": ""
```

Oops, missed one haha
```suggestion
                "value": "AGENTQL_API_KEY"
```